browser/src/DownloadsPage/DownloadsPage.module.css

```diff
@@ -0,0 +1,8 @@
+/*
+	This CSS module is used alongside Styled Components to allow modification of a single
+	html page for smooth scrolling
+*/
+
+html {
+  scroll-behavior: smooth;
```

Comment:
I tried this with smooth and with auto.

Smooth scrolling is a cool effect! It gives a bit of context to how far up or down I'm jumping on the page.

On the other hand It felt a bit janky when jumping from the very top the very bottom. Not sure how to describe it. Without smooth scroll, it felt much faster to jump to different datasets. I know that's boring though!

Could be my personal preference though... I'd suggest seeking feedback from others.